### PR TITLE
Properly celebrate the merge

### PR DESCRIPTION
**Pull request by m1guelpf**
As people watching the Klin merge stream commented on, the moment where we finally merge should have a lil more celebration 🙃

Other suggestions for what to say are also welcome. Could also add some extra whitespace for visual separation.

<details>
<summary>(note)</summary>
Really really appreciate everyone's insane work on this, and sorry for the time it'll take to review this. Keep up the good work <3

</details>

### CODE REVIEW

**Review by sporkspatula — COMMENTED** [review]
LGTM

**Review by sporkspatula — APPROVED** [review]

**Review by gtspencer — APPROVED** [review]

**Review by scotato — APPROVED** [review]

**Comment by Hugo0** [comment]
Lmao

At least delete one of the logs, it's redundant to have both

**Comment by staringispolite** [comment]
Would also have accepted

< log.Info("Left PoW stage")
> log.Info("Exit PoW, stage left")

**Comment by odyslam** [comment]
```
          .         .                                                                                             
         ,8.       ,8.          8 8888888888   8 888888888o.      ,o888888o.    8 8888888888   8 888888888o.      
        ,888.     ,888.         8 8888         8 8888    `88.    8888     `88.  8 8888         8 8888    `^888.   
       .`8888.   .`8888.        8 8888         8 8888     `88 ,8 8888       `8. 8 8888         8 8888        `88. 
      ,8.`8888. ,8.`8888.       8 8888         8 8888     ,88 88 8888           8 8888         8 8888         `88 
     ,8'8.`8888,8^8.`8888.      8 888888888888 8 8888.   ,88' 88 8888           8 888888888888 8 8888          88 
    ,8' `8.`8888' `8.`8888.     8 8888         8 888888888P'  88 8888           8 8888         8 8888          88 
   ,8'   `8.`88'   `8.`8888.    8 8888         8 8888`8b      88 8888   8888888 8 8888         8 8888         ,88 
  ,8'     `8.`'     `8.`8888.   8 8888         8 8888 `8b.    `8 8888       .8' 8 8888         8 8888        ,88' 
 ,8'       `8        `8.`8888.  8 8888         8 8888   `8b.     8888     ,88'  8 8888         8 8888    ,o88P'   
,8'         `         `8.`8888. 8 888888888888 8 8888     `88.    `8888888P'    8 888888888888 8 888888888P' 
```

**Comment by karalabe** [comment]
Though if the merge fails *after* printing a yuge celebratory message, that's gonna be an epic self burn :P

**Comment by odyslam** [comment]
```
log.warn("                                                                
8 8888888888   8 8888      88     ,o888888o.    8 8888     ,88' 
8 8888         8 8888      88    8888     `88.  8 8888    ,88'  
8 8888         8 8888      88 ,8 8888       `8. 8 8888   ,88'   
8 8888         8 8888      88 88 8888           8 8888  ,88'    
8 888888888888 8 8888      88 88 8888           8 8888 ,88'     
8 8888         8 8888      88 88 8888           8 8888 88'      
8 8888         8 8888      88 88 8888           8 888888<       
8 8888         ` 8888     ,8P `8 8888       .8' 8 8888 `Y8.     
8 8888           8888   ,d8P     8888     ,88'  8 8888   `Y8.   
8 8888            `Y88888P'       `8888888P'    8 8888     `Y8. 
")
```

File: consensus/merger.go [hunk 1]
```diff
@@ -73,6 +73,7 @@ func (m *Merger) ReachTTD() {
 	}
 	rawdb.WriteTransitionStatus(m.db, blob)
 	log.Info("Left PoW stage")
+	log.Info("WE JUST MERGED LFGGGGGGGGGGGGG")
```
Comment (holiman) [inline consensus/merger.go:76]: Meh. This isn't classy, IMO. 
I'd rather go for something more subtle. Some unicode emoticon, or maybe some quote, or maybe some fine ascii art.

Comment (MariusVanDerWijden) [reply consensus/merger.go:76]: 🐼🐼🐼

Comment (MariusVanDerWijden) [reply consensus/merger.go:76]: "A small step for a man, a huge leap for mankind"
"The internet of money should not cost $0.05 per transaction - @vbuterin"
"Ethereum ESG ETF incoming"
"I don't believe a second, compatible implementation of Bitcoin will ever be a good idea - Satoshi Nakomoto"

**Comment by karalabe** [comment]
Ok, this discussion is going nowhere fast. Going to close this and we'll see if we'll do something about it or not. Thanks for the suggestion though!

**Pull request closed without merging**